src/main/java/frc/robot/turret/commands/TurretCommands.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.turret.commands;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import frc.robot.RobotContainer;
+import frc.robot.turret.TurretState;
+import frc.robot.turret.TurretSubsystem;
+import frc.utils.joysticks.SmartJoystick;
+
+public abstract class TurretCommands {
+
+    public static Command manualControl(SmartJoystick joystick) {
+        return new FunctionalCommand(
+                () -> RobotContainer.TURRET.setState(TurretState.MANUAL),
+                () -> RobotContainer.TURRET.setPower(joystick.getAxisValue(SmartJoystick.Axis.RIGHT_X)),
```

Comment:
Another minor point but have we stopped configuring all the key bindings inside `OI.java`? What if I want this to use the Y axis instead of X?

I'd try to find a nicer way to do this.

Reply:
i'll change that to a `DoubleSupplier`.